src/helpers/AvroToMetaModel.ts

```diff
@@ -0,0 +1,435 @@
+import {
+  AnyModel,
+  ArrayModel,
+  AvroSchema,
+  BooleanModel,
+  EnumModel,
+  EnumValueModel,
+  FloatModel,
+  IntegerModel,
+  MetaModel,
+  MetaModelOptions,
+  ObjectModel,
+  ObjectPropertyModel,
+  StringModel,
+  UnionModel
+} from '../models';
+import { Logger } from '../utils';
+
+function getMetaModelOptions(AvroModel: AvroSchema): MetaModelOptions {
+  const options: MetaModelOptions = {};
+
+  if (AvroModel.const) {
+    options.const = {
+      originalInput: AvroModel.const
+    };
+  }
+  if (Array.isArray(AvroModel.type) && AvroModel.type.includes('null')) {
+    options.isNullable = true;
+  } else {
+    options.isNullable = false;
+  }
+  if (AvroModel.discriminator) {
+    options.discriminator = {
+      discriminator: AvroModel.discriminator
+    };
+  }
+  if (AvroModel.format) {
+    options.format = AvroModel.format;
+  }
```

Comment:
Dont think avro has format either.

Reply:
May be I misunderstood the `ModelOptions`. From the code, I concluded that these `keys` like `discriminator`, `format`, `const`, etc. are required to get the final Meta Model and not related to the `JSON Schema` format.
I will make the necessary changes as I got my mistake.

Reply:
Yea, they are definitely only optional 👍